app/pages/lab/visibility.cjsx

```diff
@@ -7,7 +7,7 @@ SetToggle = require '../../lib/set-toggle'
 uniq = require 'lodash/uniq'
 Paginator = require '../../talk/lib/paginator'
 
-`import ApplyForBetaForm from './apply-for-beta-form';`
+ApplyForBetaForm = require('./apply-for-beta').default
```

Comment:
Not blocking, but I don't remember why it's a good idea to remove backticks from `.cjsx`.

Reply:
They break the tests. I think it might be a bug in `coffee-react` but I've never looked into it.